crates/router/src/connector/mollie/transformers.rs

```diff
@@ -153,6 +177,55 @@ impl TryFrom<&api_models::payments::BankRedirectData> for PaymentMethodData {
     }
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MollieCardTokenRequest {
+    card_holder: Secret<String>,
+    card_number: CardNumber,
+    card_cvv: Secret<String>,
+    card_expiry_date: Secret<String>,
+    locale: String,
+    testmode: bool,
+    profile_token: String,
+}
+
+impl TryFrom<&types::TokenizationRouterData> for MollieCardTokenRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::TokenizationRouterData) -> Result<Self, Self::Error> {
+        match item.request.payment_method_data.clone() {
+            api_models::payments::PaymentMethodData::Card(ccard) => {
+                let auth = MollieAuthType::try_from(&item.connector_auth_type)?;
+                let card_holder = ccard.card_holder_name.clone();
+                let card_number = ccard.card_number.clone();
+                let card_expiry_date =
+                    ccard.get_card_expiry_month_year_2_digit_with_delimiter("/".to_owned());
+                let card_cvv = ccard.card_cvc;
+                // let locale = Secret::new("en_US".to_string());
+                let browser_info = get_browser_info(item)?;
+                let locale = browser_info
+                    .ok_or(errors::ConnectorError::MissingRequiredField {
+                        field_name: "browser_info.language",
+                    })?
+                    .language;
+                let testmode = true;
```

Comment:
This may not always be the case though?

Reply:
@swangi-kumari is this test-mode toggling decided?